services/repository-managers/pom.xml

```diff
@@ -230,6 +230,15 @@
 			<groupId>org.bouncycastle</groupId>
 			<artifactId>bcprov-ext-jdk15on</artifactId>
 		</dependency>
+
+		<!--In the "wild", images may not always be properly formatted. This is a plugin to ImageIO that allows a more lenient reading of JPEG files.
+		https://stackoverflow.com/questions/18351583/illegalargumentexception-numbers-of-source-raster-bands-and-source-color-space/18400559

```

Comment:
looked it up and this is not a commonly used plugin.  It only fixes one known image, so might not be worth the risk.